src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,329 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: ImageCapture is an API for controlling camera settings and taking photos.
```

Comment:
I'd left those two as-is (given the context) but I think you're right :). Changed.